snapshot/pkg/cloudprovider/providers/openstack/openstack_snapshots.go

```diff
@@ -69,7 +69,7 @@ func (snapshots *SnapshotsV2) createSnapshot(opts SnapshotCreateOpts) (string, s
 
 	createOpts := snapshotsV2.CreateOpts{
 		VolumeID:    opts.VolumeID,
-		Force:       false,
+		Force:       opts.Force,
```

Comment:
Why is force required?  Is it because you have to force a snapshot of a volume that is already mounted?

Reply:
Nevermind, I see your comment below:

 	// ForceSnapshotLabel - Force a snapshot to be created even if the disk is in-use.